command/operator_diagnose_test.go

```diff
@@ -320,6 +320,18 @@ func TestOperatorDiagnoseCommand_Run(t *testing.T) {
 				},
 			},
 		},
+		{
+			"diagnose_seal_transit_ok",
```

Comment:
I am still working on this, it panics!

Reply:
hmm, bad path to the config file cause vault to panic! Wondering if that should be resolved!